interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.html

```diff
@@ -1,6 +1,10 @@
-<ion-tab-bar *ngFor="let disasterType of disasterTypes">
-  <ion-tab-button (click)="switchDisasterType(disasterType)">
-    <ion-icon name="warning"></ion-icon>
-    <ion-label>{{ disasterType.label }}</ion-label>
+
+<ion-tab-bar *ngFor="let disasterType of disasterTypes"
+  [ngClass]="{'active': disasterType.label === selectedDisasterType}">
+  <ion-tab-button (click)="switchDisasterType(disasterType)" [ngClass]="{'active': disasterType.label === selectedDisasterType}">
```

Comment:
Is this conditional class an accidental duplicate? I don't see the need for two conditional classes which make the same check.

Reply:
Did it intentionally.